noxfile.py

```diff
@@ -28,7 +28,7 @@
 BLACK_PATHS = ["docs", "pybigquery", "tests", "noxfile.py", "setup.py"]
 
 DEFAULT_PYTHON_VERSION = "3.8"
-SYSTEM_TEST_PYTHON_VERSIONS = ["3.9"]
+SYSTEM_TEST_PYTHON_VERSIONS = ["3.8", "3.9"]
```

Comment:
Maybe add a comment why this library specifies two Python versions?
(since it's not about Python itself, but about different SQLAlchemy version constraints in separate files)

Reply:
Done